packages/core/src/utils/StorageUtil.ts

```diff
@@ -4,6 +4,7 @@ import type { WcWallet } from './TypeUtil.js'
 // -- Helpers -----------------------------------------------------------------
 const WC_DEEPLINK = 'WALLETCONNECT_DEEPLINK_CHOICE'
 const W3M_RECENT = '@w3m/recent'
+const W3M_IMAGE_URL = '@3m/image_url'
```

Comment:
Let's be more expressive here, which image is it (connected wallet logo / dapp logo?).